cidrgrep.go

```diff
@@ -0,0 +1,31 @@
+package cidrgrep
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"regexp"
+)
+
+// Filter filters input for lines containing an IP within cidr
+func Filter(input io.Reader, output io.Writer, cidr string) {
+	scanner := bufio.NewScanner(input)
+
+	_, ipv4Net, err := net.ParseCIDR(cidr)
```

Comment:
If you're using the blank identifier to ignore something, it's always worth saying what you're ignoring, and why, for the benefit of the reader. So we could say here:

```go
// don't need IP address, just network address
_, ipv4Net, err := net.ParseCIDR(cidr)
```

Reply:
Thanks man, that makes a lot of sense. Fixed in latest iteration